packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -46,6 +46,12 @@ const filters: Record<string,any> = {
   "meta": {
     meta: true
   },
+  "untagged": {
+    tagRelevance: {}
```

Comment:
Did you test this? I think that when something has no tags, typically the tagRelevance field will be missing, rather than the empty object. Also I don't know what mongodb will interpret an empty object to mean, in this context.

Reply:
I tested it (initially I actually was comparing it to null, which didn't work. "{}" seemed to be the thing that actually worked the way I expected, after 30 minutes of checking various alternatives and checking what was in the DB.)

Reply:
I'm not at all confident that it's performant or that this is the proper way to do it, but it is the way that worked.

Reply:
(although if you think anything would have changed with the recent softfiltering branch I can test it again to be sure)